dashboard/app.py

```diff
@@ -0,0 +1,30 @@
+from dash import Dash, dcc, html
+import plotly.express as px
+import pandas as pd
+import psycopg2
+import psycopg2.extras
+import os
```

Comment:
looks fine, however, remove these imports unless they will be used in the future